docs/documentation.rst

```diff
@@ -166,7 +166,24 @@ defaced subject:
 
    Example of manual defacing.
 
+Defacing using R
```

Comment:
should flow naturally with previous section where you talk about defacing-- the flow is illogical as implemented here-- previous section says: mri_deface doesn't work, so we recommend to do it manually-- and then this paragraph appears-- please work on the transition/overall logic of the sectinons

Reply:
fixed in https://github.com/sct-pipeline/spine-generic/pull/56/commits/aa153c59aef0156f98d5bc0c355d05ae66f2bb4a